crates/miden-objects/src/note/note_tag.rs

```diff
@@ -415,17 +394,20 @@ mod tests {
         }
 
         for account_id in network_accounts {
-            let tag = NoteTag::from_account_id(account_id, NoteExecutionMode::Local)
-                .expect("Tag generation must work with local execution and network account ID");
-            assert!(!tag.is_single_target());
-            assert_eq!(tag.execution_mode(), NoteExecutionMode::Local);
+            let tag = NoteTag::from_account_id(account_id);
+            assert!(tag.is_single_target());
+            assert_eq!(tag.execution_mode(), NoteExecutionMode::Network);
 
             tag.validate(NoteType::Public)
                 .expect("local execution should support public notes");
```

Comment:
I don't think we are testing local execution here any more - so, at the very least the `expect()` message should be updated. But also, may be good to add a comment about what exactly we are trying to test here.

This could also apply to the two loops above.